### PR TITLE
tests added

### PR DESCRIPTION
**Pull request by zvvysotskaya**
The tests work

### CODE REVIEW

File: app/Note.php [hunk 1]
```diff
@@ -13,4 +13,9 @@ class Note extends Model {
 	public function notebook() {
 		return $this->belongsTo(Notebook::class);
 	}
+
+	public function path()
```
Comment (xy2z) [inline app/Note.php]: This should be `static`, and it's missing a phpdoc

**Comment by xy2z** [comment]
Thanks! :+1: 
I will do a quick code review tomorrow

File: app/Note.php (hunk 1 above)
Comment (zvvysotskaya) [reply app/Note.php]: Thank you. Documentation is important. I am sorry; I’ll add it.

File: app/Notebook.php [hunk 2]
```diff
@@ -16,4 +18,14 @@ public function notes() {
 	public static function get_last_sort_order(int $user_id) {
 		return static::where('user_id', $user_id)->max('sort_order');
 	}
+
+	public function user()
+	{
+		return $this->belongsTo(User::class);
+	}
+
+	public function path()
```
Comment (xy2z) [inline app/Notebook.php]: should be static and needs phpdoc

File: app/Notebook.php [hunk 3]
```diff
@@ -16,4 +18,14 @@ public function notes() {
 	public static function get_last_sort_order(int $user_id) {
 		return static::where('user_id', $user_id)->max('sort_order');
 	}
+
+	public function user()
```
Comment (xy2z) [inline app/Notebook.php]: needs phpdoc

File: app/User.php [hunk 4]
```diff
@@ -36,4 +37,9 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function notebooks()
```
Comment (xy2z) [inline app/User.php]: needs phpdoc

File: database/factories/NotebookFactory.php [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+use App\Notebook;
+use App\User;
+use Faker\Generator as Faker;
+
+$factory->define(Notebook::class, function (Faker $faker) {
+    return [
+        'title' => $faker->sentence(4),
+        'sort_order' => $faker->randomDigit,
+        'user_id' => function(){
+         return factory(User::class)->create()->id;
```
Comment (xy2z) [inline database/factories/NotebookFactory.php]: needs indentation

File: tests/Feature/NoteTest.php [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+     /** @test */
+    public function an_unauthorized_user_cannot_access_a_note()
+    {
+        $note = factory(Note::class)->create();
+        $this->post('/note/create', $note->toArray())
+        ->assertRedirect('/login');
+    }
+
+    /** @test */
+    public function perm_delete_Note()
```
Comment (xy2z) [inline tests/Feature/NoteTest.php]: all method names should be lowercase ("Note" = "note")

File: tests/Feature/NotebookTest.php [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    /** @test */
+    public function an_unathorized_user_cannot_create_a_notebook()
+    {
+        $notebook = factory(Notebook::class)->create();
+        $this->post('/notebook/create', $notebook->toArray())
+        ->assertRedirect('/login');
+    }
+
+    /** @test */
+    public function delete_Notebook()
```
Comment (xy2z) [inline tests/Feature/NotebookTest.php]: all method names should be lowercase ("Notebook" = "notebook")

**Comment by xy2z** [comment]
Code-review done, just some very minor things.

**Comment by xy2z** [comment]
Thanks!

**Pull request merged**